tests/drivers/build_all/pwm/testcase.yaml

```diff
@@ -5,77 +5,58 @@ tests:
   drivers.pwm.gecko.build:
     platform_allow: efr32_radio_brd4250b
     tags: pwm_gecko
-    extra_args: "CONFIG_PWM_GECKO=y"
   drivers.pwm.imx.build:
     platform_allow: colibri_imx7d_m4
     tags: pwm_imx
-    extra_args: "CONFIG_PWM_IMX=y"
   drivers.pwm.litex.build:
     platform_allow: litex_vexriscv
     tags: pwm_litex
-    extra_args: "CONFIG_PWM_LITEX=y"
   drivers.pwm.mcux.ftm.build:
     platform_allow: frdm_k22f
     tags: pwm_mcux_ftm
-    extra_args: "CONFIG_PWM_MCUX_FTM=y"
   drivers.pwm.mcux.pwt.build:
     platform_allow: twr_ke18f
     tags: pwm_mcux_pwt
-    extra_args: "CONFIG_PWM_MCUX_PWT=y CONFIG_PWM_CAPTURE=y"
+    extra_args: "CONFIG_PWM_CAPTURE=y"
```

Comment:
why is this using `extra_args` instead of `extra_configs`?

Reply:
I was leaving it how it was before.  Changing it to be `extra_configs`